app/src/main/java/com/example/safehopper/models/Contact.java

```diff
@@ -0,0 +1,41 @@
+package com.example.safehopper.models;
+
+public class Contact extends Person {
+
+    private boolean textAlert;
+    private boolean emailAlert;
+
+    public Contact(String firstName, String lastName, String phoneNumber, String email, boolean textAlert, boolean emailAlert) {
+        super(firstName, lastName, phoneNumber, email);
+        this.textAlert = textAlert;
+        this.emailAlert = emailAlert;
+    }
+
+    public boolean isTextAlert() {
```

Comment:
Can you change this and isEmailAlert to sendTextAlert and sendEmailAlert. Those names define these functions a little better.